Index: app/src/main/java/com/example/mealer_project/ui/signup/SignupActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.mealer_project.ui.signup;\r\n\r\nimport androidx.appcompat.app.AppCompatActivity;\r\n\r\nimport android.os.Bundle;\r\n\r\nimport com.example.mealer_project.R;\r\n\r\npublic class SignupActivity extends AppCompatActivity {\r\n\r\n    @Override\r\n    protected void onCreate(Bundle savedInstanceState) {\r\n        super.onCreate(savedInstanceState);\r\n        setContentView(R.layout.activity_signup);\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/mealer_project/ui/signup/SignupActivity.java b/app/src/main/java/com/example/mealer_project/ui/signup/SignupActivity.java
--- a/app/src/main/java/com/example/mealer_project/ui/signup/SignupActivity.java	
+++ b/app/src/main/java/com/example/mealer_project/ui/signup/SignupActivity.java	
@@ -3,7 +3,7 @@
 import androidx.appcompat.app.AppCompatActivity;
 
 import android.os.Bundle;
-
+import android.widget.EditText;
 import com.example.mealer_project.R;
 
 public class SignupActivity extends AppCompatActivity {
@@ -13,4 +13,36 @@
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_signup);
     }
+
+    public String[] getInformation() {
+        // Get text from text fields
+        EditText textFirst = (EditText)findViewById(R.id.signupFirstName);
+        String firstName = textFirst.getText().toString();
+
+        EditText textLast = (EditText)findViewById(R.id.signupLastName);
+        String lastName = textLast.getText().toString();
+
+        EditText textEmail = (EditText)findViewById(R.id.signupEmailAddress);
+        String email = textEmail.getText().toString();
+
+        EditText textPassword = (EditText)findViewById(R.id.signupPassword);
+        String password = textPassword.getText().toString();
+
+        EditText textConfirmPassword = (EditText)findViewById(R.id.signupConfirmPassword);
+        String confirmPassword = textConfirmPassword.getText().toString();
+
+        EditText textAddress = (EditText)findViewById(R.id.signupStreetAddress);
+        String address = textAddress.getText().toString();
+
+        EditText textCity = (EditText)findViewById(R.id.signupCity);
+        String city = textCity.getText().toString();
+
+        EditText textPostalCode = (EditText)findViewById(R.id.signupPincode);
+        String postalCode = textPostalCode.getText().toString();
+
+        EditText textCountry = (EditText)findViewById(R.id.signupCountry);
+        String country = textCountry.getText().toString();
+
+        return new String[] {firstName, lastName, email, password, confirmPassword, address, city, postalCode, country};
+    }
 }
\ No newline at end of file
